lib/ansible/modules/utilities/logic/debug.py

```diff
@@ -1,62 +1,66 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright 2012 Dag Wieers <dag@wieers.com>
+# Copyright: (c) 2012 Dag Wieers <dag@wieers.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['stableinterface'],
                     'supported_by': 'core'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: debug
 short_description: Print statements during execution
 description:
-     - This module prints statements during execution and can be useful
-       for debugging variables or expressions without necessarily halting
-       the playbook. Useful for debugging together with the 'when:' directive.
-     - This module is also supported for Windows targets.
-version_added: "0.8"
+- This module prints statements during execution and can be useful
+  for debugging variables or expressions without necessarily halting
+  the playbook.
+- Useful for debugging together with the 'when:' directive.
+- This module is also supported for Windows targets.
+version_added: '0.8'
```

Comment:
double or single quotes?

Reply:
I don't care, I usually standardize on single quotes if a file is not consistent.